scripts/exglobal_land_analysis_prep.sh

```diff
@@ -0,0 +1,81 @@
+#! /usr/bin/env bash
+
+################################################################################
+# exglobal_land_analysis_prep.sh
+#
+# This script processes the IMS input ascii/netcdf files onto the UFS model grid.
+# Inputs: IMS ascii/netcdf file, IMS index file (is model resolution specific) 
+#         and model restart files.
+# Outputs: file with i) IMS-derived snow cover fraction over land on UFS model 
+#         grid, and ii) snow depth derived form the IMS snow cover fraction, 
+#         using an inversion of the noah model snow depletion curve.
+#
+################################################################################
+#  Utilities
+RES=${CASE:1}
+
+#  Directories.
+export gdasapp_dir=${gdasapp_dir:-"${HOMEgfs}/sorc/gdas.cd/"}
+BKGDIR=${DATA}/bkg
+
+#  Filenames
+export IMS_IODA=${IMS_IODA:-"${HOMEgfs}/sorc/gdas.cd/build/bin/imsfv3_scf2ioda.py"}
+export CALCFIMS=${CALCFIMS:-"${HOMEgfs}/sorc/gdas.cd/build/bin/calcfIMS.exe"}
+
+export OROGTYPE="C${RES}_oro_data"
+FILEDATE=${PDY}.${cyc}0000
+DOY=$(date +%j -d "${PDY} + 1 day")
+YYYY=${PDY:0:4}
+ntiles=6
+################################################################################
+# IMS proc
+if [[ ${cyc} == "18" ]]; then
+  WORKDIR=${DATA}/obs
+  cd "${WORKDIR}" || exit 99
```

Comment:
Please don't just exit.  Please put out an error message with FATAL ERROR and then exit.  Look at other examples.

Reply:
I checked other scripts (e.g., `exglobal_atmos_sfcanl.sh`), and they just use `cd ${DATA} || exit 99`. Do you have any specific suggestion so that I can make the correct changes. Thanks.